config/envs/unit_test.py

```diff
@@ -7,7 +7,7 @@
 @configclass
 class UnitTestConfig(DefaultConfig):
     TESTING = True
-    SQLALCHEMY_DATABASE_URI = "sqlite:///:memory:"
+    SQLALCHEMY_DATABASE_URI = "postgresql://postgres:password@localhost:5432/data_store"
```

Comment:
At the moment tests run by fails very slowly if we can't reach this instance when running `pytest`. 

Can we have a check that fails a test instantly if this can't be reached? 

Or alternatively checks the connection first before running any of the tests (not sure if this is an existing pytest design pattern or if it's the wrong way of thinking about it)

Reply:
Good shout, I wonder if it can go in the fixture or something?

Reply:
Moving this to subsequent test fixtures refactor ticket just to keep things moving